assets/scripts/airport.js

```diff
@@ -1346,6 +1346,7 @@ function airport_init() {
   airport_load('sbgr', "beginner", "Aeroporto Internacional de São Paulo/Guarulhos");
   airport_load('tncm', "easy", "Princess Juliana International Airport");
   airport_load('uudd', "easy", "Moscow Domodedovo Airport");
+  airport_load('vobl', "hard", "Kempegowda International Airport Bengaluru");
```

Comment:
@indianbhaji the `bundle.js` file is actually a generated file.  It contains all of the js for the app in one file and isn't meant to be edited directly.  Now, you _could_ edit it directly, but be aware that the next time a build is run, those changes will be overwritten.  A good work around, until #712 is in place, is adding the new airport to both the `bundle.js` and `airportLoadList.js`.

What is npm?  NPM is short for Node Package Manager.  Sort of like bundler for Ruby.  It provides a means to manage packages and run scripts with those packages.  v3 is written in ES2015 (or ES6 as its also known).  Browsers don't actually understand ES2105 thus, it must be transpiled to ES5.  IT gets complicated.

Long story short, when developing on the JS one will need to run `npm run build` in order to re-build the js.  When developing airports, this will also be a necessary step until #712 is completed.  A temporary workaround would be what I mentioned above.  It is very important that those new airports get into the `airportLoadList.js` otherwise they will not load after the next build.